android/src/main/java/io/sentry/react/RNSentryTimeToDisplay.java

```diff
@@ -0,0 +1,26 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.Promise;
+
+import android.view.Choreographer;
+
+import org.jetbrains.annotations.NotNull;
+import io.sentry.SentryDate;
+import io.sentry.SentryDateProvider;
+import io.sentry.android.core.SentryAndroidDateProvider;
+
+public class RNSentryTimeToDisplay {
+
+    public void GetTimeToDisplay(Promise promise) {
```

Comment:
Since we don't have any state, this could be a static function, getting the promise and dataProvider in arguments.

Reply:
Also made `createEmitNewFrameEvent` to use the shared dateProvider